src/styles/components/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  @extend %flex-between-center;
+  &__star {
+    width: 1.6rem;
+    height: 1.6rem;
+    margin-right: 4px;
+
+    @extend %bgc-image;
+
+    background-image: url('../images/star.svg');
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &.stars--4 > .stars__star:nth-child(-n + 4) {
```

Comment:
It should be the same functionality as in the stars task, if a modifier is `--3` you should show 3 active stars as well, and so on